spec/helpers/defendant_helper_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'court_data_adaptor'
+
+RSpec.describe DefendantHelper, type: :helper do
+  describe '#defendant_link_path' do
+    subject { helper.defendant_link_path(defendant) }
+
+    let(:defendant_class) { CourtDataAdaptor::Resource::Defendant }
+    # rubocop:disable RSpec/VerifiedDoubles
+    let(:defendant) { double(defendant_class) }
```

Comment:
try using `instance_double` and adding an extra allow

Reply:
@jsugarman, I am struggling to use an instance double.

If I define defendant as 
`let(:defendant) { instance_double(CourtDataAdaptor::Resource::Defendant) }`
I get the error
`the CourtDataAdaptor::Resource::Defendant class does not implement the instance method: arrest_summons_number`

If I define defendant as
`let(:defendant) { instance_double(CourtDataAdaptor::Query::Defendant::ByReference) }`
I get the error 
`CourtDataAdaptor::Query::Defendant::ByReference class does not implement the instance method: linked?`

Basically defendant link path needs both asn and linked? and neither CourtDataAdaptor::Query::Defendant::ByReference nor CourtDataAdaptor::Resource::Defendant define both.